sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -40,11 +41,30 @@
 def _load_model(
     model_class: torch.nn.Module, state_dict_path, device, **model_params):
   model = model_class(**model_params)
-  model.to(device)
+
+  if device == torch.device('cuda') and not torch.cuda.is_available():
+    logging.warning(
+        "Specified 'GPU', but could not find device. Switching to CPU.")
```

Comment:
We could add some details where this configuration was made. How about sth like:

        "Model handler specified a 'GPU' device, but GPUs are not available. Switching to CPU.")

Reply:
Done.